recipes/glew/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GlewConan(ConanFile):
+    name = "glew"
+    description = "The GLEW library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://github.com/nigels-com/glew"
+    topics = ("conan", "glew", "opengl", "wrangler", "loader", "binding")
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {"shared": [True, False]}
```

Comment:
Why fPIC is not listed?

Reply:
IDK, but you're not the only who noticed it https://github.com/bincrafters/community/issues/1158 !